particles/Blackjack/Blackjack.arcs

```diff
@@ -2,7 +2,7 @@ meta
   namespace: arcs.tutorials.blackjack
 
 // Encodes a card from 0 to 51. -1 indicates an invalid (or joker).
-schema Card
+schema CardSchema
```

Comment:
This seems worrying. What is it conflicting with, exactly? Do you have any other ideas for how to resolve this name clash without forcing the developer to rename their schemas?

Reply:
In Blackjack there is a "Card" class that implements some utilities for cards. I think these could be turned into extension functions on the Schema class which feels like a better model.